src/table/TableWrapper.jsx

```diff
@@ -34,6 +35,16 @@ export default function TableWrapper(props) {
   const paginationHidden = constraints.active && 'paginationHidden';
   const paginationFixedRpp = selectionsAPI.isModal() || tableWidth < 400;
 
+  const localState = useRef();
+
+  useEffect(() => {
+    localState.current = {
+      totalRowSize: size.qcy,
+      page,
+      rowsPerPage,
+    };
+  }, [page, rowsPerPage]);
+
```

Comment:
Why do you need this? Can't you use the `handleChangePage` function?

Reply:
it is the way to get the latest component variable value in useEffect hook.

It involves the discussion where we want to put the listener as you mentioned above.

If it should be in the component, then I guess it is not needed.